R/create_project.R

```diff
@@ -0,0 +1,256 @@
+#' Creates a pre-populated project for DfE R
+#'
+#' @param path Path of the new project
+#' @param init_renv Boolean; initiate renv in the project.
+#' Default is set to true.
+#' @param include_structure_for_pkg Additional folder structure for
+#' package development
+#' @param create_publication_proj Should the folder structure be for a
```

Comment:
These are Boolean as well I think & needed for it to run, just like init_renv

Reply:
Yep, good spot, will change these. I have set `include_structure_for_pkg()` and `create_publication_proj()` to default as `FALSE` but left `include_github_gitignore()` as open, as think it would be good practice for the user to decide this?